src/pages/Match/Participants.tsx

```diff
@@ -0,0 +1,66 @@
+import { MinusCircleOutlined } from '@ant-design/icons'
+import { FormListFieldData, InputNumber } from 'antd'
+import { Space, Form, Input, Button } from 'antd-mobile'
+
+interface ParticipantsProps {
+  fields: FormListFieldData[]
+  add: () => void
+  remove: (index: number) => void
+}
+
+export const Participants = ({ fields, add, remove }: ParticipantsProps) => {
+  return (
+    <>
+      {fields.map(({ key, name, ...restField }, index) => {
+        return (
```

Comment:
Small detail here: you can remove the curly braces "{}" and use parenthesis directly "()" after the arrow of this function, and then you don't need the `return` word